cmd/server.go

```diff
@@ -49,9 +52,33 @@ to quickly create a Cobra application.`,
 		log.Infof("Starting server on %v with path %v", listeningAddress, metricsEndpoint)
 
 		pm := phpfpm.PoolManager{}
+		// Enable dynamic pod tracking if the flag is set
+		if k8sAutoTracking {
+			log.Info("Kubernetes auto-tracking enabled. Watching for pod changes...")
+			if namespace == "" {
+				log.Info("Kubernetes namespace not set; setting to default.")
+				namespace = "default" // fallback to 'default' if not set
+			}
```

Comment:
I think that it would be more useful to default to the namespace that the exporter is deployed at. This will reduce the required configuration and make sense at the same time.

Reply:
Hey @dstrants, I actually changed the logic to either provide a namespace or default to all namespaces. 
So whenever you have the time take another look.